parflow_data_management/transport/models/authorized_key.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.db.models import CharField
+
+from scheduler.models.cluster import Cluster
+
+
+# A cluster and a valid public key for a user.
+class AuthorizedKey(models.Model):
+    cluster = models.ForeignKey(Cluster, on_delete=models.CASCADE, related_name=authorized_keys)
+    public_key = CharField(_("Public key for cluster"), max_length=4096)
```

Comment:
This will be a list on public keys?

Reply:
Each row is a cluster id and a public key for accessing that cluster. There can be multiple rows for the same cluster id if there are multiple keys for accessing it.

Reply:
sorry, yes, your right ...